.github/workflows/update-sdk.yml

```
@@ -13,23 +13,18 @@ jobs:
       - uses: actions/setup-go@v4
         with:
           go-version-file: 'go.mod'
-      - name: update
-        run: 	| 
-          go install github.com/icholy/gomajor@latest 
-          make update-atlas-sdk
+      - name: Update files
+        run:  make tools update-atlas-sdk
       - name: Verify Changed files
-        uses: tj-actions/verify-changed-files@6d688963a73d28584e163b6f62cf927a282c4d11
+        uses: tj-actions/verify-changed-files@v16
```

We do pin actions to commit. More info:
```suggestion
        uses: tj-actions/verify-changed-files@6d688963a73d28584e163b6f62cf927a282c4d11
```
https://medium.com/@wtr/best-patterns-building-github-actions-workflows-6b03655223ca

---

I see